notebooks/scrubbed/covid-19-tests-august-2020.ipynb

```diff
@@ -0,0 +1,599 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Trying to predict number of COVID-19 tests by August 1st 2020\n",
+    "*Notebook by Steven Rybicki*\n",
+    "\n",
+    "In this notebook, we'll be focusing on trying to predict the number of COVID-19 tests in the US. Namely, we'll be looking at forecasting the answer to:\n",
+    "> By August 1, how many tests for COVID-19 will have been administered in the US?\n",
+    "\n",
+    "taken from the [Metaculus question](https://pandemic.metaculus.com/questions/4400/by-august-1-how-many-tests-for-covid-19-will-have-been-administered-in-the-us/). This just looks at number of tests, not distinguishing by type or if someone is tested multiple times.\n",
+    "\n",
+    "To do this, we're going to be using [ergo](https://github.com/oughtinc/ergo), a library by [Ought](https://ought.org/). This lets you integrate model based forecasting (building up a numerical model to predict outcomes) with judgement based forecasting (using calibration to try predict outcomes less formally). To see other notebooks using ergo, see [the ergo repo](https://github.com/oughtinc/ergo/tree/master/notebooks). \n",
+    "\n",
+    "I'll be trying to walk through how you'd use this realistically if you wanted to forecast the answer to a question without spending a ton of time on it. This means our model will be kind of rough, but hopefully accurate-ish, and I'll be iteratively trying to improve it as we go along. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook setup\n",
+    "\n",
+    "This imports libraries, and sets up some useful functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install --progress-bar off poetry\n",
+    "!pip install --progress-bar off git+https://github.com/oughtinc/ergo.git@e9f6463de652f4e15d7c9ab0bb9f067fef8847f7"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import ssl\n",
+    "warnings.filterwarnings(action=\"ignore\", category=FutureWarning)\n",
+    "warnings.filterwarnings(action=\"ignore\", module=\"plotnine\")\n",
+    "ssl._create_default_https_context = ssl._create_unverified_context"
```

Comment:
That's fair. I'm going to leave this as is, since it works and is in line with the other notebook, and since I don't see us updating this anytime soon.

Reply:
Will consider this in future notebooks!